MultiServer.py

```diff
@@ -175,6 +175,7 @@ class Context:
     all_item_and_group_names: typing.Dict[str, typing.Set[str]]
     all_location_and_group_names: typing.Dict[str, typing.Set[str]]
     non_hintable_names: typing.Dict[str, typing.Set[str]]
+    spheres: typing.List[typing.Dict[int, typing.Set[int]]]
```

Comment:
```suggestion
    spheres: typing.List[typing.Dict[int, typing.Set[int]]]
    """ each sphere is { player: { location_id, ... } } """
```
just looking for something to make the data structure more clear - tell what the `int`s represent